Makefile

```diff
@@ -2,6 +2,7 @@ pkgs := $(shell go list ./...)
 run := .
 count := 1
 
+
 test: 
-	KNUU_TIMEOUT=120m go test -v $(pkgs) -run $(run) -count=$(count) -timeout 120m
+	go test -v $(pkgs) -run $(run) -count=$(count) 
```

Comment:
I would keep that `-timeout 120m` as the default is around `10m` I guess and it is not sufficient to deploy pods and perform a large or medium size test.

Reply:
oh sorry, i was reading that as 120s. my bad will revert.

Reply:
but on the other hand, this is for local knuu tests. 
our test suite should run in under 10m. 

for app, they have their own go test commands where they would need to set their timeout variable accordingly.

Reply:
yeah ideally. right now even one hour is not enough :)